### PR TITLE
move send signed contribution and proof to typedef client.

### PR DESCRIPTION
**Pull request by rolfyone**
Partially addresses #7762

## Documentation

- [X] I thought about documentation and added the `doc-change-required` label to this PR if updates are required.

## Changelog

- [X] I thought about adding a changelog entry, and added one if I deemed necessary.

### CODE REVIEW

File: validator/remote/src/main/java/tech/pegasys/teku/validator/remote/typedef/handlers/SendContributionAndProofsRequest.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  }
+
+  public void submit(final Collection<SignedContributionAndProof> signedContributionAndProofs) {
+    final List<SignedContributionAndProof> requestData =
+        new ArrayList<>(signedContributionAndProofs);
+    postJson(
+        SEND_CONTRIBUTION_AND_PROOF,
+        Collections.emptyMap(),
+        requestData,
+        listOf(requestData.getFirst().getSchema().getJsonTypeDefinition()),
```
Comment (tbenr) [inline validator/remote/src/main/java/tech/pegasys/teku/validator/remote/typedef/handlers/SendContributionAndProofsRequest.java:49]: what if the collection is empty? getFirst will throw

Comment (tbenr) [reply validator/remote/src/main/java/tech/pegasys/teku/validator/remote/typedef/handlers/SendContributionAndProofsRequest.java:49]: image

it is a bit annoying to resolve with the current design.

Comment (rolfyone) [reply validator/remote/src/main/java/tech/pegasys/teku/validator/remote/typedef/handlers/SendContributionAndProofsRequest.java:49]: yep this is the same as previous, i had considered that and gone through a couple of iterations including a warning, but decided to leave the behaviour... i can check and see if we can ever call that with 0, but it has never seemed to cause us a problem so i was favouring old behaviour... will dig a tiny bit into old one, not sure we'd ever send signed with 0 (my initial code just returned)

**Review by lucassaldanha — APPROVED** [review]
LGTM

File: validator/remote/src/main/java/tech/pegasys/teku/validator/remote/typedef/handlers/SendContributionAndProofsRequest.java (hunk 1 above)
Comment (rolfyone) [reply validator/remote/src/main/java/tech/pegasys/teku/validator/remote/typedef/handlers/SendContributionAndProofsRequest.java:49]: hopefully that warning doesnt spam - it shouldnt - it should be a bug if we get that far.

**Pull request merged**